data-model/src/main/java/io/micronaut/data/annotation/Relation.java

```diff
@@ -56,6 +56,10 @@
          * Cascade insert operations.
          */
         PERSIST,
+        /**
+         * Cascade merge operations.
+         */
+        MERGE,
```

Comment:
Merge is not really supported with JDBC?

Reply:
I need to distinguish persist/update. It looked like a good idea to use the same name. Do you think it's better to have UPDATE or not having an extra operation at all?

Reply:
probably `UPDATE` is more appropriate a name